backend/routers/finance.py

```
@@ -141,11 +142,48 @@ async def get_finance_application(application_id: str, request: Request):
-        owner_uid_filter = None if has_read_all else caller_uid
+        owner_uid_filter = caller_uid
+
+        if has_read_all:
+            try:
+                can_override = rbac_manager.can_admin_or_expert_override(
+                    ctx,
+                    resource_owner_uid=None,
+                    resource_tenant_id=resource_tenant_id,
+                    allow_cross_tenant=False,
+                )
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)